src/components/Form/Picklist.js

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import enhanceWithClickOutside from 'react-click-outside';
+
+import {
+  PicklistDropdown,
+  PicklistDropdownList,
+  PicklistDropdownListItem,
+  InputRaw,
+} from '../../';
+
+export class Picklist extends React.Component {
+
+  constructor(props, context) {
+    super(props, context);
+    this.state = { isOpen: false };
```

Comment:
For increased readability, add a blank line before this statement.

Reply:
👍